app/src/org/commcare/android/adapters/MenuAdapter.java

```diff
@@ -87,9 +85,7 @@ public MenuAdapter(Context context, CommCarePlatform platform, String menuID){
                         try {
```

Comment:
I think we need to move this try/catch up to the whole scope, btw, to catch the menu relevancy expressions too